primefaces/src/main/java/org/primefaces/component/fileupload/FileUploadRenderer.java

```diff
@@ -64,12 +65,6 @@ public void decode(FacesContext context, UIComponent component) {
     public void encodeEnd(FacesContext context, UIComponent component) throws IOException {
         FileUpload fileUpload = (FileUpload) component;
 
-        if (fileUpload.getDropZone() != null) {
```

Comment:
We are back porting a breaking change which is not even documented?

Reply:
@tandraschko is this a breaking change?  I thought this was only ADDING functionality to SimpleFileUpload?

Reply:
Oh I see previously if you had a DROPZONE it assumed you wanted Advanced mode.

Reply:
OK in Main we would need to add the same note in the 14.0.0 Migration Guide because adding it here won't do any good the docs are only built from Main!

Reply:
We need to add this to a 14_x migration guide

Reply:
Yep i will take care of it